Experimental/README.md

```diff
@@ -1,22 +1,29 @@
 # ‼️ Experimental Samples! Use at your own risk ‼️
 
-The goal of this folder is to aggregate ideas and small projects to be shared with the community, without guaranteeing those are Unity recommended practices. They are there for fostering discussion and with little quality guarantee, not for production use.
+The Experimental folder is to collect and share ideas and small projects among the Unity Multiplayer community. These samples are not guaranteed to follow Unity's recommended practices. They are specifically for fostering discussion and little quality guarantee; they are not for production use.
 
-This code might get promoted to full bitesize samples or they also might get removed if we figure they are not promoting good practices anymore. Please be critical when looking at these samples and use these at your own risk. If you need education for game features critical to your production, don't use these samples and stick to our regular samples and docs.
+These code samples may become full Bitesize samples or may be removed they are no longer promoting good, up-to-date practices.
```

Comment:
mmm @MorwennaG and I updated the branch this morning, some of your changes might be on outdated changes :(

Reply:
actually, took a look at the diff between Morwenna's proposal and yours, most of your changes are outdated now. We moved the release process to an external doc and cleaned up the initial section and added more details.